ci-operator/step-registry/hypershift/openstack/create/hypershift-openstack-create-chain.yaml

```diff
@@ -0,0 +1,208 @@
+chain:
+  as: hypershift-openstack-create
+  steps:
+  - as: create-hostedcluster
+    cli: latest
+    env:
+    - name: HYPERSHIFT_NODE_COUNT
+      default: "3"
+      documentation: "The number nodes to automatically create and join to the cluster."
+    - name: HYPERSHIFT_BASE_DOMAIN
+      default: "origin-ci-int-aws.dev.rhcloud.com"
```

Comment:
So this has to come with the AWS credentials that are used later in the dnsrecords provision step. I had issues in the past when using the shiftstack subdomain and I think that's because the management cluster is on AWS and the credential used there allow me to manage records in `origin-ci-int-aws.dev.rhcloud.com`.

We can re-iterate on that later but since this is easily overridable I think we can leave it here for now, if you're ok.

Reply:
yeah the Route53 credentials are embedded in the cluster_profile secret